### PR TITLE
add archwaytestnetwork

### PR DESCRIPTION
**Pull request by jlehtimaki**
This PR adds Archways test network called constantine-1

### CODE REVIEW

File: testnets/archwaytestnet/assetlist.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "constantine-1",
```
Comment (JeremyParish69) [inline testnets/archwaytestnet/assetlist.json:3]: please change this chain_name to `archwaytestnet` the chain_id will still be retained in the chain.json file.

File: testnets/archwaytestnet/chain.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "archwaytestnet",
+  "chain_id": "constantine-1",
+  "pretty_name": "Archway testnet",
+  "status": "live",
+  "network_type": "testnet",
+  "website": "https://archway.io",
+  "bech32_prefix": "archway",
```
Comment (JeremyParish69) [inline testnets/archwaytestnet/chain.json:9]: Please register this bech32 prefix tp https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Comment (jlehtimaki) [reply testnets/archwaytestnet/chain.json:9]: Done: https://github.com/satoshilabs/slips/pull/1510

File: testnets/archwaytestnet/assetlist.json (hunk 1 above)
Comment (jlehtimaki) [reply testnets/archwaytestnet/assetlist.json]: Fixed

**Pull request merged**